### DISCUSSION TITLE
[Feature] can user change the server url without logout?

### DISCUSSION THREAD

**Discussion by xuxingcan** [immich-app/immich#14937] [Feature Request]
### I have searched the existing feature requests to make sure this is not a duplicate request.

- [X] Yes

### The feature

Sometimes there may be a server address change, but the server content has not changed. However, if you want to reconnect in this situation, you need to log out and log in again, but every time you log in, the image will be re uploaded to the server, even if it is the same. If only we could directly change the URL

### Platform

- [ ] Server
- [ ] Web
- [X] Mobile

**Comment by bo0tzz** [comment]
#2335
